runtime/CSharp/src/Tree/Xpath/XPathLexer.cs

```diff
@@ -46,7 +43,7 @@ public const int
 	};
 
 	public static readonly string[] ruleNames = {
-		"Anywhere", "Root", "Wildcard", "Bang", "ID", "NameChar", "NameStartChar",
+		"Anywhere", "Root", "Wildcard", "Bang", "ID", "NameChar", "NameStartChar", 
```

Comment:
Excess whitespace at the end on the line (or changed line break?).

Reply:
weird.  this is generated code though so...